docs/porting/third-party-deps.md

```diff
@@ -0,0 +1,50 @@
+# Analyzing your 3rd Party Dependencies
+
+By [Phillip Carter](https://github.com/cartermp)
+
+The first step in the porting process is to understand your third party dependencies, figure out which of them (if any), don't run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
```

Comment:
hyphen third-party and remove the comma after (if any). Also which don't run on .NET Core seems a bit repetitive in this paragraph. Maybe you can simplify the sentence to something like "figure out if there's any that doesn't run on .NET Core and develop a contingency plan for them."